app/views/evaluation_forms/_form.html.erb

```diff
@@ -33,10 +36,14 @@
         <%= form.text_field :challenge_id %>
       </div>
 
-      <div>
-        <%= form.label :instructions, style: "display: block" %>
-        <%= form.text_field :instructions %>
+      <div class="usa-character-count">
```

Comment:
looks like this works 👍 can we add the other character counters? in addition to the FE validation, we should probably add the backend schema limits and model validations too.